pipdeptree.py

```diff
@@ -469,7 +478,7 @@ def reverse(self):
         return PackageDAG(dict(m))
 
 
-def render_text(tree, list_all=True, frozen=False):
+def render_text(tree, list_all=True, frozen=False, output=None):
```

Comment:
Please update docstring with new arg. Also, I suggest making it a mandatory arg. The caller can pass a file-like object explicitly.